src/routes/AgreementViewRoute.js

```diff
@@ -220,6 +222,11 @@ class AgreementViewRoute extends React.Component {
     this.props.history.push(`${urls.agreements()}${this.props.location.search}`);
   }
 
+  handleClickFilterButton = (path) => {
```

Comment:
I think `handleFilterEResources` might be more explanatory of what we're doing with this filter. An agreement is so big so it's helpful to have pretty specific names for our callbacks/props.